src/components/CodeInput/styled/index.js

```diff
@@ -77,8 +81,12 @@ const StyledInput = attachThemeAttrs(styled.input)`
         :focus,
         :active {
             box-shadow: none;
-            background-color: ${props.palette.background.main};
-            border: 1px solid ${props.palette.border.main};
+
+            ${props.renderActiveInputStyles &&
+                `
+                background-color: ${props.palette.background.main};
+                border: 1px solid ${props.palette.border.main};
+            `}
```

Comment:
@blakmetall I think if you put the isActive block before the Error block, the problem is solved

Reply:
sure, I improved now based on your hint

Reply:
sure, logic improved based on your hint